packages/client/hmi-client/src/router/index.ts

```diff
@@ -7,12 +7,14 @@ import SimulationPlanPlayground from '@/views/SimulationPlanPlayground.vue';
 import TheiaView from '@/views/theia.vue';
 import DocumentView from '@/views/document.vue';
 import Simulation from '@/views/Simulation.vue';
+import ProjectView from '@/components/projects/Project.vue';
 import Model from '@/views/Model.vue';
 
 export enum RoutePath {
 	Home = '/',
 
 	DocView = '/docs/:id?',
+	Project = '/projects/:id',
```

Comment:
Maybe use :projectId to be more consistent, e.g. so we can do 
```
$router.params.projectId
```
and not worry about if id = project or something else